Misc/NEWS.d/next/Library/2022-05-21-20-25-04.gh-issue-93033.QuMGyh.rst

```diff
@@ -0,0 +1 @@
+Use wmemchr in stringlib where applicable when the size of STRINGLIB_CHAR equals the size of wchar_t. All the places wmemchr is added where places memchr was previously used when STRINGLIB_CHAR equaled size of char.
```

Comment:
It can go in the commit message, but the common Python user does not have any idea what are stringlib and STRINGLIB_CHAR. Please rewrite the NEWS entry in Python terms. What is the effect of the change for the Python programmer and how large is it.

Reply:
How does:
"String wide-character string operations include 'find` with a needle of length one and `replace` with a needle and replacement of length one may be sped up." sound?

Reply:
Wide-character is not a thing in Python.

Something like: "Search in some strings (with maximal character in the range U+0100 to U+FFFF on Windows or astral containing characters on other platforms) is now up to 1000 times faster."

Reply:
Thanks. Created new blurb. Will that auto-update?

Reply:
You could just update this file instead of adding new file. Please do it next time.